next.config.js

```diff
@@ -63,6 +63,18 @@ const nextConfig = {
         basePath: false,
         permanent: false,
       },
+      {
+        source: `${basePath}/uninstalled/`,
+        destination: 'https://forms.gle/2q33AzpTA3A6vvZo6',
+        basePath: false,
+        permanent: true,
+      },
+      {
+        source: `${basePath}/first-tab/`,
+        destination: `${basePath}/`,
```

Comment:
This won't work without an absolute URL. See [docs](https://nextjs.org/docs/api-reference/next.config.js/rewrites#rewrites-with-basepath-support) and failed deploy.

Regardless, you can set `basePath` to false and exclude it from the source and destination.